framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -65,21 +65,33 @@ int cfg_get_param(const std::string &param, std::string &value)
     return RETURN_ERR;
 }
 
-int cfg_get_param_int(const std::string &param, int &value)
+static int cfg_get_prplmesh_param_int(const std::string &param, int default_value)
```

Comment:
This should not change at all - each API on its own should take care of the default value which is used only if reading from the DB fails.